<launch>
	<include file="$(find ras_line_detector)/launch/line_detector.launch"/>
	<node name="particle_filter" type="particle_filter" pkg="ras_particle_filter"/>

	<param name="predict_freq" type="int" value="10" />
	<param name="update_freq" type="int" value="5" />
	<param name="start_pose" value="[0.0, 0.0, 0.0]" />
	<param name="Lambda_psi" type="double" value="0.001" />
	<param name="kp_left" type="double" value="0.3" />
	<param name="M" type="int" value="1000" />
	<param name="R" value="[0.01, 0.01, 0.01]" />
	<param name="Q" value="[0.01, 0.01]" />
	<param name="part_bound" type="double" value="0.0" />
	<param name="bound" value="[0.0, 1.0, 0.0, 1.0]" />
	<param name="n_particles_to_pub" type="int" value="1" />

</launch>
